### ISSUE TITLE
RadioExt not working under Proton...?

### ISSUE THREAD

**Issue by 0x5066** [justarandomguyintheinternet/CP77_radioExt#10]
Have just tried the newest version, and it seems that it won't run properly under Proton, despite having all the necessary dependencies installed.

`[RadioExt] Error: Red4Ext part of the mod is missing`

The log file just consists of these messages over and over:

```
[2023-10-09 00:25:56 UTC+02:00] [320] init.lua:73: attempt to index field 'radioManager' (a nil value)
stack traceback:
	init.lua:73: in function <init.lua:67>
[2023-10-09 00:25:56 UTC+02:00] [320] init.lua:73: attempt to index field 'radioManager' (a nil value)
stack traceback:
	init.lua:73: in function <init.lua:67>
[2023-10-09 00:25:56 UTC+02:00] [320] init.lua:73: attempt to index field 'radioManager' (a nil value)
stack traceback:
	init.lua:73: in function <init.lua:67>
[2023-10-09 00:25:56 UTC+02:00] [320] init.lua:73: attempt to index field 'radioManager' (a nil value)
stack traceback:
	init.lua:73: in function <init.lua:67>
[2023-10-09 00:25:56 UTC+02:00] [320] init.lua:73: attempt to index field 'radioManager' (a nil value)
stack traceback:
	init.lua:73: in function <init.lua:67>
[2023-10-09 00:25:56 UTC+02:00] [320] init.lua:64: attempt to index field 'radioManager' (a nil value)
stack traceback:
	init.lua:64: in function <init.lua:63>
```

This is the error I keep getting, what should I do?

**Comment by BillyBobJones678** [comment]
same here

**Comment by BillyBobJones678** [comment]
fixed it you have to update your red4ext

**Comment by 0x5066** [comment]
That's what I did, numerous times over just to make sure I have all the latest files

**Comment by gajama** [comment]
Get v2.7 from Nexus: [https://www.nexusmods.com/cyberpunk2077/mods/4591?tab=files](https://www.nexusmods.com/cyberpunk2077/mods/4591?tab=files)

That version works with Proton for me.

It seems this repo often lags behind the Nexus releases (a hint I got looking at some of the closed issues).

**Comment by Zen79050** [comment]
This issue still persists

**Comment by gajama** [comment]
With the latest version (2.7) downloaded from Nexus?

**Comment by luigiMinardi** [comment]
> With the latest version (2.7) downloaded from Nexus?

Yes, I have the latest (2.7) and it says the same thing on the CET console.

**Comment by 0x5066** [comment]
That's weird, when I tried this through winecfg, nothing launched anymore.
I think I'll try this again some time.

**Comment by FossPrime** [comment]
> > With the latest version (2.7) downloaded from Nexus?
> 
> ~Yes, I have the latest (2.7) and it says the same thing on the CET console.~
> 
> Solved here by adding `WINEDLLOVERRIDES="winmm,version=n,b" %command%` to my launcher options at steam as per the [documentation](https://wiki.redmodding.org/cyberpunk-2077-modding/for-mod-users/users-modding-cyberpunk-2077/modding-on-linux#tl-dr).

I needed this on Proton 11... Worth a mention on the the install instructions IMO

**Comment by luigiMinardi** [comment]
> I needed this on Proton 11... Worth a mention on the the install instructions IMO

It's mentioned on the docs of the official modding tool by cdprojektred for modding cyberpunk on the linux part of it. Might be worth mentioning that you do need to have redmod correctly configured for it to work and where is its docs but most mods will assume you already had this knowledge.
